test/util/util.go

```diff
@@ -531,8 +531,8 @@ func ExpectLQByStatusMetric(lq *kueue.LocalQueue, status metav1.ConditionStatus)
 		if metrics.ConditionStatusValues[i] == status {
 			wantV = 1
 		}
-		metric := metrics.LocalQueueByStatus.WithLabelValues(lq.Name, lq.Namespace, string(s))
 		gomega.EventuallyWithOffset(1, func(g gomega.Gomega) {
+			metric := metrics.LocalQueueByStatus.WithLabelValues(lq.Name, lq.Namespace, string(s))
```

Comment:
/hold

Let me check something.

Reply:
I think it’s much better to wrap the entire `for` loop with `gomega.EventuallyWithOffset`.